multus_sriov_cni_install.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export MULTUS_CNI_BRANCH=${MULTUS_CNI_BRANCH:-master}
+export MULTUS_CNI_REPO=${MULTUS_CNI_REPO:-https://github.com/intel/multus-cni}
+
+export SRIOV_CNI_BRANCH=${SRIOV_CNI_BRANCH:-master}
+export SRIOV_CNI_REPO=${SRIOV_CNI_REPO:-https://github.com/intel/sriov-cni}
+
+export PLUGINS_BRANCH=${PLUGINS_BRANCH:-master}
+export PLUGINS_REPO=${PLUGINS_REPO:-https://github.com/containernetworking/plugins.git}
+
+export SRIOV_NETWORK_DEVICE_PLUGIN_REPO=${SRIOV_NETWORK_DEVICE_PLUGIN_REPO:-https://github.com/intel/sriov-network-device-plugin}
+export SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH=${SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH:-master}
+
+export GOROOT=${GOROOT:-/usr/local/go}
+export GOPATH=${WORKSPACE}
+export PATH=/usr/local/go/bin/:/opt/cni/bin/:$GOPATH/src/k8s.io/kubernetes/third_party/etcd:$PATH
+
+export CNI_BIN_DIR=${CNI_BIN_DIR:-/opt/cni/bin/}
+export CNI_CONF_DIR=${CNI_CONF_DIR:-/etc/cni/net.d/}
+export ALLOW_PRIVILEGED=${ALLOW_PRIVILEGED:-true}
+export NET_PLUGIN=${NET_PLUGIN:-cni}
+
+export KUBE_ENABLE_CLUSTER_DNS=${KUBE_ENABLE_CLUSTER_DNS:-false}
+export API_HOST=$(hostname).$(hostname -y)
+export API_HOST_IP=$(hostname -I | awk '{print $1}')
+export KUBECONFIG=${KUBECONFIG:-/var/run/kubernetes/admin.kubeconfig}
+
+#TODO add autodiscovering
+export MACVLAN_INTERFACE=${MACVLAN_INTERFACE:-eno1}
+export SRIOV_INTERFACE=${SRIOV_INTERFACE:-enp3s0f0}
+export VFS_NUM=${VFS_NUM:-4}
+
+echo "Working under $WORKSPACE folder"
+mkdir -p $WORKSPACE
+mkdir -p $LOGDIR
+mkdir -p $ARTIFACTS
+mkdir -p $CNI_CONF_DIR
+mkdir -p $CNI_BIN_DIR
+
+pushd $WORKSPACE
+
+
+function configure_multus {
+
+    echo "Configure Multus"
+    kubectl create -f $WORKSPACE/multus-cni/images/multus-daemonset.yml
+    sleep 60
```

Comment:
Instead of `sleep`, perhaps we could check the ds status and if the daemonset pod is running.